fred2/missionnotesdlg.cpp

```diff
@@ -314,7 +314,7 @@ void CMissionNotesDlg::OnOK()
 	pad_with_newline(m_mission_notes, NOTES_LENGTH - 1);
 
 	string_copy(The_mission.name, m_mission_title, NAME_LENGTH - 1, 1);
-	string_copy(The_mission.author, m_designer_name, NAME_LENGTH - 1, 1);
+	The_mission.author = m_designer_name;
```

Comment:
The `string_copy` function calls `set_modified` if the old name and new name are different.  This assignment should do the same thing.

There is a `string_copy` for C-style strings and for CStrings, but not for SCP_strings.  I'd suggest making a SCP_string version using the CString version as a basis.

Reply:
This is done

Reply:
So it is.  Cool